dashy/src/app/dashboard/widget/divider-widget/divider-widget.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="col-md-12">
+  <h2>{{title}}</h2>
```

Comment:
IMHO you should use a smaller font-size for the heading here. Your `h2` is even bigger than the `h1` title of the dashboard itself.